sonic-psud/scripts/psud

```diff
@@ -14,7 +14,7 @@ try:
     import signal
     import sys
     import threading
-
+    from datetime import datetime
```

Comment:
Why we need this?

Reply:
In the following piece of code, we update datetime.
https://github.com/Azure/sonic-platform-daemons/blob/e6c786bd02e6a253fcf67dec895e7c24e940c788/sonic-psud/scripts/psud#L557 
Since the datetime module is not imported, all the Fan fields will not be updated to DB.